scripts/build-snarkyjs-node.sh

```diff
@@ -0,0 +1,16 @@
+SNARKY_JS_PATH=$1
+[ -z "$SNARKY_JS_PATH" ] && echo "Usage: ./build-snarkyjs-node.sh [/path/to/snarkyjs]" && exit 1
+
+dune b src/lib/snarky_js_bindings/snarky_js_node.bc.js
+cp _build/default/src/lib/crypto/kimchi_bindings/js/node_js/plonk_wasm* "$SNARKY_JS_PATH"/src/node_bindings/
+cp _build/default/src/lib/snarky_js_bindings/snarky_js_node*.js "$SNARKY_JS_PATH"/src/node_bindings/
+
+# better error messages
+sed -i 's/function failwith(s){throw \[0,Failure,s\]/function failwith(s){throw joo_global_object.Error(s.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.js
```

Comment:
Wrapping with try ... catch unfortunately doesn't achieve what this does: By throwing a proper JS `Error` instead of _an array_, I ensure that the error message has an error trace attached, which hugely reduces the time needed to figure out where the error comes from.

I agree `sed` is a bad hack. The ideal solution to this and many other problems is probably to fork `js_of_ocaml` and fix it.